semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -114,6 +115,30 @@ def attributes(self):
         if not hasattr(self, "attrs_by_name"):
             return []
 
+        return list(
+            filter(
+                lambda attr: not AttributeType.is_template_type(attr.attr_type),
+                list(self.attrs_by_name.values()),
+            )
+        )
+
+    @property
+    def attribute_templates(self):
+        if not hasattr(self, "attrs_by_name"):
+            return []
+
+        return list(
+            filter(
+                lambda attr: AttributeType.is_template_type(attr.attr_type),
+                list(self.attrs_by_name.values()),
+            )
+        )
```

Comment:
Extract a helper method: `def _get_attributes(self, *, templates: bool)` that is called from `attribute_templates` and `attributes` since they only differ by checking whether is_template_type is True vs False.

Reply:
done